google/cloud/forseti/scanner/scanner.py

```diff
@@ -120,7 +122,8 @@ def run(model_name=None,
             except Exception:  # pylint: disable=broad-except
                 log_message = 'Error running scanner: {}'.format(
                     scanner.__class__.__name__)
-                progress_queue.put(log_message)
+                progress_queue.put('{}: \'{}\''.format(
```

Comment:
Instead of using `.format()` a second time, can you combine this into one?

Something like this:
```
log_message = 'Error running scanner: {}: {}'.format(
    scanner.__class__.__name__,
    traceback.format_exc())
progress_queue.put(log_message)
```

Reply:
Fixed- thanks!